src/register/components/ConfigurableRegistrationForm.jsx

```diff
@@ -33,7 +33,10 @@ const ConfigurableRegistrationForm = (props) => {
     autoSubmitRegistrationForm,
   } = props;
 
-  const countryList = useMemo(() => getCountryList(getLocale()), []);
+  // The reason for adding the entry 'United States' is that sometime browser aut-fill the form with the 'Unites States'
+  // instead 'United States of America' which does not exists in country dropdown list and gets the user confused
+  // and unable to create account. So we added the United States entry in dropdown list.
```

Comment:
```suggestion
 /** The reason for adding the entry 'United States' is that Chrome browser aut-fill the form with the 'Unites 
  States' instead of 'United States of America' which does not exist in country dropdown list and gets the user 
  confused and unable to create an account. So we added the United States entry in the dropdown list.
 */
```